iothub_client/inc/iothub_client_core_common.h

```diff
@@ -150,14 +150,80 @@ extern "C"
 
     MU_DEFINE_ENUM_WITHOUT_INVALID(DEVICE_TWIN_UPDATE_STATE, DEVICE_TWIN_UPDATE_STATE_VALUES);
 
+/**
+ * @brief Current versions of @p IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS structure.
+ */
+#define IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_STRUCT_VERSION_1 1
```

Comment:
IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS_VERSION_1 ?

Reply:
Using the same approach John is for PnP. Struct versioning required in case it grows later, all due to dynamic library support.

https://github.com/Azure/azure-iot-sdk-c/blob/01f0d50e23fa2a8dbd16f4e3508fa18acce7e5c0/iothub_client/inc/iothub_client_properties.h#L122